packages/client/types/constEnums.ts

```diff
@@ -369,7 +369,7 @@ export const enum Threshold {
   MAX_POKER_DIMENSION_NAME = 50,
   MAX_REACTJIS = 12,
   MAX_POKER_STORIES = 50,
-  MAX_GITLAB_POKER_STORIES = 15,
+  MAX_GITLAB_POKER_STORIES = 50, // TODO: remove this and use MAX_POKER_STORIES instead
```

Comment:
I'll remove this and refactor in a separate PR to keep this PR as small as possible